owasm/src/ext/flight/openskynetwork.rs

```diff
@@ -0,0 +1,119 @@
+        let mut members = parsed.members();
+        if members.len() == 0 {
+            return Some(false);
+        }
+        members.nth(0)?["icao24"].as_str()?;
+        Some(
+            members
+                .filter(|x| x["icao24"].as_str() == Some(&self.icao24))
+                .collect::<Vec<&json::JsonValue>>()
+                .len()
+                > 0,
+        )
+    }
+}
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn test_as_cmd() {
+        assert_eq!(
+            Flight::new("a79de9",&FlightOption::DEPARTURE, "EDDF", 1000, 900).as_cmd(),
+            ShellCmd::new("curl", &["https://opensky-network.org/api/flights/departure?airport=EDDF&begin=900&end=1000"])
+        );
+    }
+
+    #[test]
+    fn test_from_cmd_found_flight_ok() {
+        assert_eq!(
+            Flight::new("a79de9",&FlightOption::DEPARTURE, "EDDF", 1580937440,1580997440).from_cmd_output(r#"[{"icao24":"406459","firstSeen":1580938171,"estDepartureAirport":"EDDF","lastSeen":1580941248,"estArrivalAirport":"EDDP","callsign":"BCS30C ","estDepartureAirportHorizDistance":2588,"estDepartureAirportVertDistance":194,"estArrivalAirportHorizDistance":7435,"estArrivalAirportVertDistance":132,"departureAirportCandidatesCount":1,"arrivalAirportCandidatesCount":6},{"icao24":"a79de9","firstSeen":1580937606,"estDepartureAirport":"EDDF","lastSeen":1580941006,"estArrivalAirport":"LFPG","callsign":"FDX36","estDepartureAirportHorizDistance":2854,"estDepartureAirportVertDistance":187,"estArrivalAirportHorizDistance":2411,"estArrivalAirportVertDistance":157,"departureAirportCandidatesCount":2,"arrivalAirportCandidatesCount":8},{"icao24":"3c5426","firstSeen":1580937574,"estDepartureAirport":"EDDF","lastSeen":1580940348,"estArrivalAirport":"EDDP","callsign":"BCS27H ","estDepartureAirportHorizDistance":2669,"estDepartureAirportVertDistance":194,"estArrivalAirportHorizDistance":4686,"estArrivalAirportVertDistance":25,"departureAirportCandidatesCount":2,"arrivalAirportCandidatesCount":6}]"#.into()),
+            Some(true)
+        );
+    }
+
+    #[test]
+    fn test_from_cmd_not_found_flight_ok() {
+        assert_eq!(
+            Flight::new("a79de9",&FlightOption::DEPARTURE, "EDDF", 1580937440,1580997440).from_cmd_output(r#"[{"icao24":"406459","firstSeen":1580938171,"estDepartureAirport":"EDDF","lastSeen":1580941248,"estArrivalAirport":"EDDP","callsign":"BCS30C ","estDepartureAirportHorizDistance":2588,"estDepartureAirportVertDistance":194,"estArrivalAirportHorizDistance":7435,"estArrivalAirportVertDistance":132,"departureAirportCandidatesCount":1,"arrivalAirportCandidatesCount":6},{"icao24":"3c5426","firstSeen":1580937574,"estDepartureAirport":"EDDF","lastSeen":1580940348,"estArrivalAirport":"EDDP","callsign":"BCS27H ","estDepartureAirportHorizDistance":2669,"estDepartureAirportVertDistance":194,"estArrivalAirportHorizDistance":4686,"estArrivalAirportVertDistance":25,"departureAirportCandidatesCount":2,"arrivalAirportCandidatesCount":6}]"#.into()),
```

Comment:
Between 2 number and comma why not have whitespace?

Reply:
done